test/boxplot-labels.html

```diff
@@ -0,0 +1,256 @@
+                                                emphasis: {
+                                                    show: true,
+                                                    position: 'top',
+                                                    formatter: 'zxcv'
+                                                }
+                                            }
+                                        },
+                                        {
+                                            name: 'max value (default)',
+                                            type: 'max'
+                                        },
+                                        {
+                                            name: 'min value (default)',
+                                            type: 'min'
+                                        },
+                                        {
+                                            name: 'max value (dim:Q1)',
+                                            type: 'max',
+                                            valueDim: 'Q1'
+                                        },
+                                        {
+                                            name: 'average value (dim:Q1)',
+                                            type: 'average',
+                                            valueDim: 'Q1'
+                                        }
+                                    ]
+                                },
+
+                                markLine: {
+                                    data: [
+                                        [
+                                            {name: '两个坐标之间的标线', coord: [1, 240]},
+                                            {coord: [2, 260]}
+                                        ],
+                                        [
+                                            {name: '两个屏幕坐标之间的标线', x: 50, y: 60},
+                                            {x: 70, y: 90}
+                                        ],
+                                        [
+                                            {name: 'max - min', type: 'max'},
+                                            {type: 'min'}
+                                        ],
+                                        {
+                                            name: 'min line',
+                                            type: 'min'
+                                        },
+                                        {
+                                            name: 'max line on dim:Q3',
+                                            type: 'max',
+                                            valueDim: 'Q3'
+                                        }
+                                    ]
+                                },
+                                label: {
+                                    show: true,
+                                    formatter: function (param) {
```

Comment:
If `formatter` is not provided here, there will be an error. You should not assert it to be defined.